browser/net/BUILD.gn

```diff
@@ -7,7 +7,10 @@ import("//build/config/features.gni")
 defines = brave_service_key_defines
 
 source_set("net") {
-  configs += [ "//brave/build/geolocation" ]
+  configs += [
+    "//brave/build/geolocation",
+    "//brave/components/brave_component_updater/browser:network_config",
```

Comment:
this would stay as-is with https://github.com/brave/brave-core/pull/5854/files#r442516329 because this doesn't have or need a dep on `//brave/components/brave_component_updater/browser`